news/2021-09-14-skinners-bulletin-july-2021.md

```diff
@@ -0,0 +1,122 @@
+
+Inside the circle-clicking boundaries, the skin remains relatively player-friendly and straightforward, which exposes the magnificent design of the health bar at the top-left corner. Bordering almost all the elements, including the spinner, dashed and dotted lines boosted the aesthetic a little alongside the clouds. The purple glow preferred for the skip screen combined with the Baal artwork amplifies the concept thanks to its similarities with the Balethunder phenomenon, which Genshin Impact enthusiasts would fall for. As for the pause and fail menus, even though full-screen artwork use tends to knock off the quality of the skin, Raiden Shogun manages to rub them with the menu interface very well.
+
+Last but not least, Raiden Shogun - Baal is one of the perfect examples of how character-orientation and minimalism should be mixed into one. Nonetheless, we are but the messenger, and deciding on the skin that goes well with your taste falls onto your hands, so give its showcase video a go before making a decision:
+
+<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/FhY1ikQljiU" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+And that wraps up our skin showcases for July 2021! We'll come back with more of them in the forthcoming issues, but until then, have a look around the rest of the bulletin to enlighten yourself with everything skinning.
+
+## News & Tips
+
+Curious about what is going on with the skinning community? Read all about it right away!
+
+### Completed Skins Compendium Rework
+
+Previously consisting of a long list of skins grouped under 5 prominent categories, the [Completed Skins Compendium](https://osu.ppy.sh/community/forums/topics/686664) has been pulled out of forums and replaced by [a fully operational website](https://compendium.skinship.xyz/ "skinship") on its own. Thanks to its advanced sorting options coupled with the search bar, the new compendium ensures that none of the skins require browsing further than skin-deep!
+
+Although the website is being hosted outside of osu!, the content is based on the [Completed Skins subforum](https://osu.ppy.sh/community/forums/109), meaning that the skins should be wholly original and safe-for-work as per the [rules of the Skinning forum](https://osu.ppy.sh/community/forums/topics/318193).
```

Comment:
Should be sounds kind of uncertain, while any skin that doesn't follow the forum rules also won't make it into the compendium. Unless the rules part only applies to the NSFW statement and you want to imply that skins might not be wholly original.

```suggestion
Although the website is being hosted outside of osu!, the content is based on the [Completed Skins subforum](https://osu.ppy.sh/community/forums/109), meaning that the skins are wholly original and safe-for-work as per the [rules of the Skinning forum](https://osu.ppy.sh/community/forums/topics/318193).
```